backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js

```diff
@@ -26,9 +25,11 @@ const {
 	viewGroups,
 } = require('../../../../../schemas/tickets/templates.constants');
 const { createResponseCode, templates } = require('../../../../../utils/responseCodes');
+const { deleteGroups, getGroupsByIds } = require('../../../../../models/tickets.groups');
```

Comment:
this starts to  look confusing as to why we're using model here but processor for addGroups. I think we should just have  the functions (just a passthrough, assign the model functions to the processor object) in processors so it's getting all the group functions in one file.